CHANGELOG.md

```diff
@@ -7,10 +7,12 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 21.38.6
```

Comment:
Nit pick - I think that this is a minor upgrade rather than a patch. Happy either way though.

Reply:
Agree with Ian, as we're introducing a new feature, otherwise it's good to go.